### ISSUE TITLE
Unable to configure vrang.bp.unrelease.versions / UNRELEASE_BLUEPRINT_VERSIONS in config / installer

### ISSUE THREAD

**Issue by jskirde** [vmware/build-tools-for-vmware-aria#364]
### Description

When trying to prevent cloud templates from being "unreleased", we can affect the release locally using vrang.bp.unrelease.versions but this cannot be set when doing package releases.  As a result, when a release is performed all previous versions of a template are "unreleased". 
Why is this important to us... it's because the VRA Terraform provider attempts to query the specific cloud template version and if it's missing, results in config drift that is inescapable. We need to retain all cloud template versions and they need to be published.

### Steps to Reproduce

1. Make a change to a cloud template
2. Run some code to package and release... something like:
```
mvn release:prepare --batch-mode --settings ${SETTINGS} -DreleaseVersion=$ver -Dresume=false -DscmCommentPrefix='Release ' -DpushChanges=false
mvn release:perform --batch-mode --settings ${SETTINGS} -Pbundle-with-installer -Darguments="-Dinclude='${include}'" -Dvrang.bp.unrelease.versions='false'
unzip "vRA8Catalog/${TAG}/vRA8Catalog-${TAG}-bundle.zip"
chmod +x ./bin/installer
./bin/installer ${props_file} 
```
# .. where props_file is the path to the properties file that will be used to influence the deployment. This is where we would expect to be able to set `UNRELEASE_BLUEPRINT_VERSIONS`.

**Preconditions:** [What are the preconditions to reproduce the issue]
- You have an existing cloud template with at least one version published

**Expected behavior:** [What you expect to happen]
- I expect to be able to stop the build tools from unreleasing all versions. This was somewhat the intent of https://github.com/vmware/build-tools-for-vmware-aria/issues/104

**Actual behavior:** [What actually happens]
- build tools unreleases all versions except the latest when doing a prepare/release.

**Reproduces how often:** [What percentage of the time does it reproduce]
100%

**Component/s:** [What are the Build Tools for VMware Aria components affected by the issue (e.g. "common/artifact-manager", "maven/plugins/vra-ng", "typescript/vrotest", etc)]

**Affects Build/s:** [Which are the Build Tools for VMware Aria releases / builds affected by the issue]
2.34+

#### Client

-   Build Tools for VMware Aria Version:
-   Visual Studio Code Version:
-   OS Version:

#### Server

-   vRealize Automation Version:
-   vRealize Orchestrator Version:
-   vRealize Operations Version:
-   vRealize Log Insight Version:

### Related issues and PRs

https://github.com/vmware/build-tools-for-vmware-aria/issues/104

**Comment by Michaelpalacce** [comment]
Good catch @jskirde seems like I missed to add this functionality to the Installer
